spec/lib/extensions/database_configuration_spec.rb

```diff
@@ -11,14 +13,20 @@
 
   context "ERB in the template" do
     before do
-      database_config = StringIO.new "---\nvalue: <%= 1 %>\n"
-      allow(database_config).to receive(:exist?).and_return(true)
-      allow(Pathname).to receive(:new).and_return(database_config)
+      @database_config = Tempfile.new
+      @database_config.write("---\nvalue: <%= 1 %>\n")
+      @database_config.rewind
+
+      allow(File).to receive(:expand_path).and_return(@database_config.path)
```

Comment:
This line worries me a bit because we also call `File.expand_path` in the spec definition of `db_config_path`
I think it would be better to stub this as we do below which here would be:

```ruby
expect(Pathname).to receive(:new).with(db_config_path).and_return(Pathname.new(@database_config.path))
```

Reply:
Ok, done.